src/layouts/HomeLayout.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <q-layout view="hHh lpR fFf">
+
+    <q-header elevated class="bg-primary text-white" height-hint="98">
+      <q-toolbar>
+        <q-btn dense flat round icon="menu" @click="toggleLeftDrawer" />
+
+        <q-toolbar-title>
+          <q-avatar>
+            <!--image goes here<img src="https://cdn.quasar.dev/logo-v2/svg/logo-mono-white.svg">-->
+          </q-avatar>
+          Salisbury Wiki IT
+        </q-toolbar-title>
+      </q-toolbar>
+
+      <q-tabs align="left">
+        <q-route-tab to="/Login" label="Login" />
+        <q-route-tab to="/About" label="About" />
+        <q-route-tab to="/page3" label="Student" />
+      </q-tabs>
+    </q-header>
+
+    <q-drawer show-if-above v-model="leftDrawerOpen" side="left" bordered>
+      <LeftDrawer/>
+    </q-drawer>
+
+    <q-page-container>
+      <router-view />
```

Comment:
<router-view></router-view> is already in App.vue. Switch this with <slot></slot> and then you can import the layout and pass in the view that you want.

Example, you could reuse this for the login page:
```
<home-layout>
  <Login/>
</home-layout>
```

Reply:
I would like to correct my comment here as I was wrong. I did some more reading (links below). This router-view is fine if you go with the router setup that quasar recommends & the other team is using. 

Basically, this nested router-view acts as a mount point for sub-routes or child components defined in the route. This is a perfectly fine paradigm and if you want to use it, go ahead. 

Here is the documentation on it. https://quasar.dev/layout/routing-with-layouts-and-pages

Reply:
@B-Ryan4 lol